lib/cogctl/actions/groups/create.ex

```diff
@@ -1,36 +1,28 @@
 defmodule Cogctl.Actions.Groups.Create do
   use Cogctl.Action, "groups create"
-  alias Cogctl.Table
+
+  alias Cogctl.Actions.Groups
+  alias CogApi.HTTP.Client
 
   def option_spec do
-    [{:name, :undefined, 'name', {:string, :undefined}, 'Group name (required)'}]
+    [{:name, :undefined, :undefined, {:string, :undefined}, 'Group name (required)'}]
   end
 
-  def run(options, _args, _config, endpoint) do
-    with_authentication(endpoint,
-                        &do_create(&1, :proplists.get_value(:name, options)))
+  def run(options, _args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, nil, nil, &1))
   end
 
-  defp do_create(_endpoint, :undefined) do
-    display_arguments_error
+  def run(options, _args, _config, endpoint) do
+    do_create(endpoint, :proplists.get_value(:name, options))
```

Comment:
It was broken out to be consistent with the other group actions. I kind of hate the with_authentication wrapper and if we follow this pattern we get to ignore it since the run function gets called the same way every time.